src/Microsoft.Health.Dicom.Functions/Indexing/Configuration/ReindexConfiguration.cs

```diff
@@ -39,5 +41,13 @@ public class ReindexConfiguration
         ///// </summary>
         //[Range(0d, 1d)]
         //public double MaxErrorPercentage { get; set; } = 1;
+
+        public bool IsPurgeErrorEnabled { get; set; } = true;
+
+        public IEnumerable<OrchestrationRuntimeStatus> RuntimeStatus { get; } = new[] { OrchestrationRuntimeStatus.Completed };
```

Comment:
This should probably be some sort of readonly collection, like `IEnumerable<T>`, but also have the `Count` proeprty available such as `IReadOnlyCollection<T>`. No reason to expose this as the "lazily" evaluated interface `IEnumerable<T>`

Reply:
is this what you mean?
`public IReadOnlyCollection<OrchestrationRuntimeStatus> RuntimeStatus { get; } = new[] { OrchestrationRuntimeStatus.Unknown };`

Reply:
You'll want to keep `set` in there too, otherwise I don't think the binder will be able to set it